django/contrib/auth/views.py

```diff
@@ -178,10 +178,16 @@ def logout(request, *args, **kwargs):
 
 
 @deprecate_current_app
-def logout_then_login(request, login_url=None, extra_context=None):
+def logout_then_login(request, login_url=None, **kwargs):
```

Comment:
It would be better to use `extra_context=_sentinel` similar to what was done in 9a30acad8a1996c914351bad981d937de4db29a4.